src/components/StatusButton.tsx

```diff
@@ -1,17 +1,17 @@
-import Link from "next/link";
-
 type StatusType = {
-  link: string;
+  changeStatus: () => Promise<void>;
```

Comment:
これだとだめなんですかね？
button onClick には普通Promiseの関数を渡さないと思うので
あと、他のファイルでエラー出さないためにダミーの関数定義されてますが、それならこっちに ? つけておけば解決するのでは

```suggestion
  changeStatus?: () => void;
```

Reply:
@yuto-ono 
確かにそうでした！
https://github.com/tatata05/next-app/pull/22/commits/960cb3004b344f206354c04f6a19c5d936f6309b
修正いたしました。
合わせて、ShiftAbsence コンポーネントも調整しました。